code/widgets/org.eclipse.scout.widgets.ui.html/src/main/js/forms/hybrid/PersonJsForm.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010, 2023 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import {CheckBoxField, DateField, Form, FormModel, InitModelOf, models, StringField} from '@eclipse-scout/core';
+import PersonJsFormModel, {PersonJsFormWidgetMap} from './PersonJsFormModel';
+import {PersonDo} from '../../index';
+
+export class PersonJsForm extends Form {
+  declare widgetMap: PersonJsFormWidgetMap;
+  declare data: object;
+
+  nameField: StringField;
```

Comment:
Please do not save the fields as members. This is not necessary. constructor() can be removed as well.